### PR TITLE
add simpleIdentifier to 1.2-dev

### LINKED ISSUE #815: Add Tender.identifier(s) and Contract.identifier(s) fields
https://github.com/gitMHOCDS/dncp-contract-code adds contracts.dncpContractCode (string)

> For each contract, the DNCP generates a contract code that differs from the id of the contractId it self, and this information is relevant because it helps users to link a contract in Hacienda with a contract in DNCP. Users are very used to working with this information in Paraguay, and as such it is needed for the Paraguayan users of the data.

https://github.com/openprocurement/ocds_contractNumber_extension adds contracts.contractNumber (string)

> This extension introduces one field that can be used to provide editable number of the contract.

The earlier discussion in #289 provides additional examples.

Rather than the approach in #289 – which is for each publisher to create their own very similar but different extension – the proposal here is to add to `Contract` either a `identifier` field for a single `Identifier`, or an `identifiers` field for an array of `Identifier`.

The two local extensions suggest only `identifier` is needed.

cc @yolile @duncandewhurst @myroslav

### PR DESCRIPTION
**Pull request by odscjen**
closes #815 and resolves #1488

### CODE REVIEW

File: schema/release-schema.json
Comment (duncandewhurst) [inline schema/release-schema.json]:
```suggestion
          "title": "ID",
```

**Review by duncandewhurst — CHANGES_REQUESTED** [review]
@jpmckinney I think the tests need updating so that `id` not being required on `SimpleIdentifier` doesn't cause a failure.

**Review by duncandewhurst — APPROVED** [review]

File: schema/release-schema.json
Comment (jpmckinney) [inline schema/release-schema.json]: Change word agreement, since this is in the context of one identifier.

```suggestion
          "description": "The list, register or system from which the identifier is taken.",
```

**Comment by jpmckinney** [comment]
> @jpmckinney I think the tests need updating so that `id` not being required on `SimpleIdentifier` doesn't cause a failure.

What should the merge behavior be? For `Organization.additionalIdentifiers`, we use `"wholeListMerge": true,`. If we were to add that here, then the tests will not error.

If we want to merge by identifier, then I think `id` needs to be required.

**Comment by duncandewhurst** [comment]
Ah, good point. I think `wholeListMerge` makes sense, since `SimpleIdentifier.id` need only be unique within the scope of `.scheme`.

**Comment by jpmckinney** [comment]
I've made that changed and resolved my comment. I just need one approval before merge. Note my commit has some extra changes in the dereferenced file, as pre-commit hadn't been run in some earlier commit.

**Review by duncandewhurst — APPROVED** [review]

**Pull request merged**
